Dockerfile

```diff
@@ -23,4 +25,4 @@ ENV keymap=default
 
 VOLUME /qmk
 WORKDIR /qmk
-CMD make clean ; make keyboard=${keyboard} subproject=${subproject} keymap=${keymap}
```

Comment:
The make script reads them from the environment vars, so they seemed redundant.  Would you like me to remove that change?